ocsptools/ocsp_tools_test.go

```diff
@@ -0,0 +1,51 @@
+package ocsptools
+
+import (
+	"github.com/googleinterns/ocsp-response-linter/linter"
+	"golang.org/x/crypto/ocsp"
+	"testing"
+)
+
+const (
+	GoodResp = "../testdata/resps/oldfbresp"
+	GoodCert = "../testdata/certs/google.der"
+	BadPath = "/blah/blah/blah"
+	// ExpiredCert = "./testdata/certs/expiredcert.der"
+	// NoIssuingURLCert = "./testdata/certs/rootcert.der"
+	// RevokedURL = "revoked.grc.com:443"
+	// BadURL = "blah.blah.blah"
```

Comment:
Do you still need these commented out bits?

Reply:
I'll be sure to clean that up in my next commit and all other instances of commented out code (except one place with mocking the linter, I have an explanation for that in the code)

Reply:
Sorry the next commit is buried in https://github.com/googleinterns/ocsp-response-linter/pull/14, which is being merged into this branch